test/worlds/__init__.py

```diff
@@ -1,20 +1,42 @@
 def load_tests(loader, standard_tests, pattern):
     import os
     import unittest
-    from ..TestBase import file_path
+    import Utils
+    import typing
+    import zipfile
+    import importlib
+    import inspect
+
     from worlds.AutoWorld import AutoWorldRegister
 
     suite = unittest.TestSuite()
     suite.addTests(standard_tests)
-    folders = [os.path.join(os.path.split(world.__file__)[0], "test")
+    folders = [(os.path.join(os.path.split(world.__file__)[0], "test"), world.zip_path)
                for world in AutoWorldRegister.world_types.values()]
 
-    all_tests = [
-        test_case for folder in folders if os.path.exists(folder)
-        for test_collection in loader.discover(folder, top_level_dir=file_path)
-        for test_suite in test_collection
-        for test_case in test_suite
+    all_tests: typing.List[unittest.TestCase] = [
     ]
 
+    for folder, zip_path in folders:
+        if os.path.exists(folder) and not zip_path:
+            all_tests.extend(
+                test_case
+                for test_collection in loader.discover(folder, top_level_dir=Utils.local_path("."))
```

Comment:
`Utils.local_path(".")` is "/usr/lib/python3.10/unittest/."  and fails. Reverting to `..TestBase.file_path` works for me.
```suggestion
                for test_collection in loader.discover(folder, top_level_dir=file_path)
```